Console/Command/GraphShell.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-App::uses('AppShell', 'Console\Command');
+App::uses('AppShell', 'Console/Command');
```

Comment:
I agree, that change is bad.
`App::uses('AppShell', 'Console/Command');` is correct and has always been that way :)